vertical-pod-autoscaler/e2e/v1/recommender.go

```diff
@@ -22,6 +22,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/onsi/ginkgo"
```

Comment:
Why move these two imports, we usually keep them after k8s.io (as a separate section)?

Reply:
No reason to do that.